src/cheats.c

```diff
@@ -214,25 +220,65 @@ static void cheat_10_minute_inspections()
 	window_invalidate_by_class(WC_RIDE);
 }
 
-static void cheat_increase_money(money32 amount)
+static void cheat_no_money(int enabled)
 {
-	money32 currentMoney;
+	if (enabled == 1) {
+		gParkFlags |= PARK_FLAGS_NO_MONEY;
```

Comment:
@duncanspumpkin  I see where you are going with this and while i agree.. right now, as it stands, the park parameters aren't synced, but are displayed in the third cheats button anyway. when it comes to using scenarios as sandboxes (which is pretty much easily achieved by own all and invent all land), it is troublesome to have to pause the map, open the park parameters from the cheats button, save the map, and reload it. and explain to people why it was paused or why everyone possibly got disconnected. it can be a bit of a hassle. not to mention having to explain to new server hosters how to do all of that. would we be able to keep this option there for now and then maybe remove it later? 
or just keep it there, as no money can be seen as a cheat as well? and if we sync the park parameters that can be accessed by the third cheats tab, we may end up with the argument "if it's a money cheat, why not have it in the money cheats window? why hide it away in the third tab in another seperate window?" that may get tossed up. much like the cricitism of RCT Worlds staffing where you have to go into the rides tab and build staff buildings. 

again, i see what you are talking about, and you make sense, but i don't think having "no money" available in the cheats window would be that much of a problem. i mean, we have a sandbox mode button in the third tab and in the cheats dropdown.